components/patient-data/ui/src/main/resources/PhenoTips/GeneClass.xml

```diff
@@ -117,7 +117,7 @@
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <values>candidate=Candidate|rejected=Negative|solved=Confirmed causal|carrier=Carrier</values>
+      <values>candidate=Candidate|rejected=Rejected candidate|solved=Confirmed causal|carrier=Carrier|negative=Negative</values>
       <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
```

Comment:
I wonder if this is the correct order in which the options are listed.

Reply:
We should ask Brittney...

Reply:
changed the order up to suggested by Brittney